app/models/course.rb

```diff
@@ -235,6 +235,28 @@ def new_draft_attributes
     where(is_send: true)
   end
 
+  scope :with_funding_types, ->(funding_types) do
+    program_types = []
+
+    if funding_types.include?("salary")
+      program_types.push :school_direct_salaried_training_programme
```

Comment:
personal preference to use shovel `<<` instead of `push` so
```
program_types << :school_direct_salaried_training_programme
```

Reply:
Sure no prob